NEWS.md

```diff
@@ -21,13 +26,13 @@ Development configuration is stored as an option named `rextendr.extendr_dev_dep
 
 * A `<pkg_name>-win.def` file containing DLL exports is created by `rextendr::use_extendr()`. It is used during linking phase on Windows and solves the problem of compiling very large projects, such as `polars` ([#212](https://github.com/extendr/rextendr/pull/212))
 
-* Support extendr macro with options (#128).
+* Support extendr macro with options ([#128](https://github.com/extendr/rextendr/pull/128)).
```

Comment:
I think it is common practice not to manually link to GitHub issue numbers because `pkgdown` and others do it automatically.